src/main/java/edu/greenblitz/robotName/OI.java

```diff
@@ -81,13 +78,9 @@ public SmartJoystick getFourthJoystick() {
 	public void initButtons() {
 		secondJoystick.START.whileTrue(new PanicMode());
 		secondJoystick.BACK.whileTrue(new ToggleScoringMode());
-		secondJoystick.A.whileTrue(new ScoreToAmp());
-		secondJoystick.B.whileTrue(new MoveElbowAndWristToSafe());
-		secondJoystick.X.whileTrue(new MoveShooterToAngle(PivotConstants.PresetPositions.PICK_UP.ANGLE));
-		secondJoystick.Y.whileTrue(new MoveShooterToAngle(PivotConstants.PresetPositions.TRANSFER.ANGLE));
 		secondJoystick.POV_DOWN.whileTrue(new CollectNoteToScoringMode());
 
-		mainJoystick.A.whileTrue(new GoToShootingState(ShootingPositionConstants.OPTIMAL_SHOOTING_ZONE));
+		mainJoystick.A.whileTrue(new ScoreToScoringMode());
```

Comment:
why in init buttons?

Reply:
score to scoring mode replaces the GoToShootingState